homeassistant/helpers/template.py

```diff
@@ -2059,6 +2067,9 @@ def wrapper(*args, **kwargs):
         self.globals["device_attr"] = hassfunction(device_attr)
         self.globals["is_device_attr"] = hassfunction(is_device_attr)
 
+        self.globals["entry_id"] = hassfunction(entry_id)
```

Comment:
We should clarify that this is a config entry id. Just `entry_id` could mean any entry id. We have many registries with entries in Home Assistant.